packages/ui-react/src/components/HeroShelf/HeroShelf.module.scss

```diff
@@ -97,6 +98,7 @@ $mobile-landscape-height: 100vh;
   height: 56.25vw;
   max-height: 700px;
   background-color: var(--hero-shelf-background-color);
+  transition: opacity ease-out 0.3s;
```

Comment:
IMO the scroll animation feels better without using an extra CSS transition (no delay). The scroll events already defines the animation timeline and also is more performance friendly, but that is probably not even noticeable.

Reply:
Thanks, I agree. Changed!